Uchu.World/Objects/GameObjects/Player.cs

```diff
@@ -276,6 +280,30 @@ public async Task<float[]> GetCollectedAsync()
             return flagValues;
         }
 
+        public async Task TriggerCelebration(int CelebrationID)
```

Comment:
Nice, once we see uses for this in other game objects we might think about moving this to an extension class but imo it's good for now